### DISCUSSION TITLE
API issue

### DISCUSSION THREAD

**Discussion by iturdzonyou** [wolveix/satisfactory-server#292] [Q&A] [answered]
Docker Compose

Okay so I read the readme. I pulled the latest image, set udp/tcp to 7777. Both ports are open on my router/firewall.

Says If you're experiencing API connectivity issues, your issue is that you haven't completed one of these two steps. I did both those steps and am still getting the API issue.

# Satisfactory
        satisfactory:
            container_name: "satisfactory-server"
            hostname: satisfactory-server
            image: 'wolveix/satisfactory-server:latest'
            ports:
                - 7777:7777/udp
                - 7777:7777/tcp
            volumes:
                - /home/media/drives/disk3/SatisfactoryDocker:/config
            environment:
                - MAXPLAYERS=12
                - PGID=1000
                - PUID=1000
                - ROOTLESS=false
                - STEAMBETA=false
                - AUTOSAVEONDISCONNECT=true
                - AUTOPAUSE=false
            restart: unless-stopped

![satisfactorydocker](https://github.com/user-attachments/assets/2e4dea07-16c0-41bb-bafd-d19d563e332a)

![satisfactoryip](https://github.com/user-attachments/assets/c61b95a2-ae0a-4cc9-bd28-13a7b469be6d)

**Comment by wolveix** [comment, marked as answer]
@skhalid93 you're either not running the latest version (make sure to manually run `docker compose pull`, alternatively change your image tag to `wolveix/satisfactory-server:v1.8`), or your system hosting the container has a firewall blocking the TCP port. If it's Ubuntu, it's probably through `ufw`.

**Reply by iturdzonyou** [reply]
I did all that but it still wasn't working. It works now though with the latest update. Thank you!
